src/main/java/org/dcsa/tnt/service/impl/TNTEventServiceImpl.java

```diff
@@ -1,20 +1,32 @@
 package org.dcsa.tnt.service.impl;
 
 import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
+import org.dcsa.core.events.model.EquipmentEvent;
 import org.dcsa.core.events.model.Event;
+import org.dcsa.core.events.model.ShipmentEvent;
+import org.dcsa.core.events.model.TransportEvent;
+import org.dcsa.core.events.repository.EventRepository;
 import org.dcsa.core.events.repository.PendingEventRepository;
+import org.dcsa.core.events.service.EquipmentEventService;
+import org.dcsa.core.events.service.GenericEventService;
+import org.dcsa.core.events.service.ShipmentEventService;
+import org.dcsa.core.events.service.TransportEventService;
 import org.dcsa.core.events.service.impl.GenericEventServiceImpl;
-import org.dcsa.tnt.service.EventService;
+import org.dcsa.core.exception.NotFoundException;
+import org.dcsa.core.extendedrequest.ExtendedRequest;
+import org.dcsa.core.service.impl.ExtendedBaseServiceImpl;
+import org.dcsa.tnt.service.TNTEventService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 import java.time.OffsetDateTime;
+import java.util.UUID;
 
-@Slf4j
 @RequiredArgsConstructor
 @Service
-public class EventServiceImpl extends GenericEventServiceImpl implements EventService {
+public class TNTEventServiceImpl extends GenericEventServiceImpl implements TNTEventService {
 
     private final PendingEventRepository pendingEventRepository;
```

Comment:
I cannot write a comment above (where it might belong) so I am writing it here.
PendingEventRepository - is this a specific TnT requirement - or is this a general Event requirement? I think it is the latter - in which case I also think that PendingEventRepository belongs to Event_Core. I might be wrong - but isn't the purpose of PendingEventRepository to manage events that have not yet been sent - this is a general requirement that we need in all projects...